docs/venv_guide.rst

```diff
@@ -7,16 +7,11 @@ Creating a virtual environment is really easy and usually prevents many common i
 problems. Firstly, simply choose how you'd like to create your virtual environment:
 
 * :ref:`using-venv` (quick and easy, involves two commands)
-* :ref:`using-pyenv-virtualenv` (recommended if you installed Python with pyenv)
-
-**Why Should I Use a Virtual Environment?**
-
-90% of the installation and setup issues raised in our support channels are resolved when the user
-creates a virtual environment.
+* :ref:`using-pyenv-virtualenv` (required if you installed Python with pyenv)
```

Comment:
It's not required to use `pyenv virtualenv` when you install Python with `pyenv`.